CSSfiles/Mstyle.css

```diff
@@ -0,0 +1,370 @@
+  }
+
+  .main-page {
+    font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
+  }
+
+  .heading-Mob {
+    padding: 20px;
+    position: relative;
+    display: grid;
+  }
+
+  .heading-Des {
+    display: none;
+  }
+
+  .logo {
+    grid-column: 1;
+    color: aliceblue;
+    color: #fff;
+    font-weight: 600;
+    text-decoration: none;
+  }
+
+  nav {
+    grid-column: 2;
+    color: rgb(255, 255, 255);
+    margin-left: auto;
+    margin-right: 0;
+  }
+
+  .D-nav {
+    display: none;
+  }
+
+  .intro {
+    color: #7f89f3;
+    text-align: center;
+    font-size: 30px;
+  }
+
+  .title {
+    color: rgb(211, 202, 202);
+    font-size: 30px;
+    text-align: center;
+    margin-top: 30px;
+  }
+
+  .about {
+    color: rgb(200, 196, 196);
+    text-align: center;
+    margin: 40px 30px;
+    font-size: 15px;
+    line-height: 25px;
+  }
+
+  .after-about {
+    position: relative;
+    width: 90%;
+    height: 1.1px;
+    background-color: rgb(39, 42, 42);
+    color: rgb(5, 8, 46);
+    align-items: center;
+    margin-left: 5%;
+  }
+
+  .languages-skills-frameworks {
+    position: relative;
+    display: grid;
+    grid-template-columns: repeat(2, 1fr);
+    grid-gap: 30px;
+    margin-top: 50px;
+    align-items: center;
+    justify-content: center;
+    left: 10%;
+    background-image: url("Ellipse 19.png");
+    background-repeat: no-repeat;
+    background-position: right;
+  }
+
+  .languages {
+    flex: 1;
+    position: relative;
+  }
+
+  .skills {
+    flex: 1;
+    position: relative;
+  }
+
+  .frameworks {
+    flex: 1;
+    position: relative;
+  }
+
+  .word {
+    color: rgb(202, 205, 205);
+    margin: 10px 0;
+    font-size: 13px;
+  }
+
+  li {
+    list-style: none;
+  }
+
+  .group {
+    position: relative;
+    color: #7f89f3;
+    bottom: 10px;
+  }
+
+  .icon-work {
+    display: flex;
+    flex-direction: column;
+  }
+
+  .icons {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    color: white;
+    font-size: 120%;
+    margin: 30px;
+    size: 150%;
+  }
+
+  .github-icon {
+    margin: 0 8px;
+  }
+
+  .linkedIn-icon {
+    margin: 0 8px;
+  }
+
+  .peace-icon {
+    margin: 0 8px;
+  }
+
+  .twitter-icon {
+    margin: 0 8px;
+  }
+
+  .M-icon {
+    margin: 0 8px;
+  }
+
+  .work-down-arrow-D {
+    display: none;
+  }
+
+  .work-down-arrow {
+    display: flex;
+    align-self: center;
+    color: rgb(163, 160, 234);
+  }
+
+  .icon {
+    padding-left: 38%;
+  }
+
+  .wor {
+    text-align: center;
+    margin-left: 10%;
+  }
+
+  .title-work {
+    color: #413cc6;
+    font-size: 150%;
+    margin-left: 5%;
+    margin-top: 80px;
+  }
+
+  .title-work-D {
+    display: none;
+  }
+
+  .line {
+    position: relative;
+    background-color: rgb(39, 42, 42);
+    height: 1px;
+    width: 90%;
+    color: rgb(3, 3, 104);
```

Comment:
- [ ] **Required:** Oops 🙈 ! Looks like the color of the splitter/line breakers is a little different from the [design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=23%3A976). Kindly check the color codes [using the inspect feature](https://help.figma.com/hc/en-us/articles/360055203533-Use-the-Inspect-panel), to help you match the exact look.